blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -114,6 +111,24 @@ public void removeAllMetricNames() {
         log.info("[TRACKER] All metric names removed.");
     }
 
+    public Set<String> getMetricNames() {
+        return this.metricNames;
```

Comment:
I don't have strong feeling about this either way.  I'll get rid of it as your preference.